bindings/java/nz/mega/sdk/MegaLoggerInterface.java

```diff
@@ -1,5 +1,36 @@
 package nz.mega.sdk;
 
+/**
+ * Interface to receive SDK logs.
+ * <p>
+ * You can implement this class and pass an object of your subclass to MegaApi.setLoggerClass() to receive SDK logs.
+ * You will also have to use MegaApi.setLogLevel() to select the level of the logs that you want to receive.
+ */
 public interface MegaLoggerInterface {
+    /**
+     * This function will be called for all logs with level <= your selected level of logging.
+     * <p>
+     * By default log level is MegaApi.LOG_LEVEL_INFO.
+     *  
+     * @param time
+     *            Readable string representing the current time.
+     *            The SDK retains the ownership of this string, it will not be valid after this function returns
+     * @param loglevel
+     *            Log level of this message.
+     *            Valid values are: <br>
+     *          • LOG_LEVEL_FATAL = 0 <br>
```

Comment:
Try to avoid unicode characters (the bullet). Use an "unordered list" (from HTML) instead, or just an asterisk (`*`) or dash (`-`) to keep it simple.